MAL-21 fixes

jeyabaltag opened the pull request:
@bruskiza I have updated the code for this issue https://praekeltorg.atlassian.net/browse/MAL-21. Also I have attached the screenshot for this report. Please review it and let me know if you have anything on this. Thanks!

![mal-21](https://cloud.githubusercontent.com/assets/7973498/17815312/5ed40172-6651-11e6-9cc6-0ab8756bd9ea.png)

Commit:
MAL-21 fixes

bruskiza commented:
Hey @jeyabaltag,

This is looking great!

Two minor adjustments please:
1. Please add a "No international travel" column for when Abroad is "No"
2. Please add a header "Group" for the international travel history as per MAL-21.
   I have included an example screenshot as reference.
   <img width="1199" alt="international_travel_history" src="https://cloud.githubusercontent.com/assets/165585/17844598/544ce106-683b-11e6-90b9-d113795dc8ae.png">

If this doesn't make sense, please send me the raw html and i will adjust directly and send back to you.

Commit:
No international travel column added

jeyabaltag commented:
Hi @bruskiza ,
I have updated the code for add a new column "No international travel" and added the new Group header. Also I have attached the screenshot.
Please review it and let me know if anything anything on this.
Thanks!

![national_travel](https://cloud.githubusercontent.com/assets/7973498/17851246/a377b9d4-687e-11e6-813f-b534081cf409.png)

bruskiza commented:
@jeyabaltag please merge develop back into this branch... it's failing.

bruskiza committed:
Pinning pytest to 2.9

bruskiza commented:
@jeyabaltag that fixed it.
👍 please merge.
